src/style.scss

```diff
@@ -150,15 +103,15 @@ slot {
   @media (hover: hover) {
     &:active,
     &:hover {
-      background-color: var(--auro-color-ui-hover-on-light);
```

Comment:
I don't remember why we made this change as part of this PR? The PR description says it was done but not why. Please clarify why the change was made in the description.

Reply:
We had to change from `background-color` to `background-image` for the hover styles in order to allow for `linear-gradient`

Reply:
These were changed from `background-color` to `background-image` in order to work together with the `linear-gradient` we wanted to put on all hover styles